src/app/staticanalysis/modules/txOrigin.js

```diff
@@ -1,4 +1,4 @@
-var name = 'tx.origin: '
+var name = 'Origin: '
```

Comment:
Hi Yann,  Yes, transaction origin, which is how it is described after the lead "Origin: ".   This change is cleaner and leaves no ambiguity about the field being tx.origin.